src/player-actions-logger/pipes/parse-filters.pipe.ts

```diff
@@ -0,0 +1,37 @@
+import { Player } from '@/players/models/player';
+import { PlayersService } from '@/players/services/players.service';
+import { Injectable, PipeTransform } from '@nestjs/common';
+import { isEmpty } from 'lodash';
+import { FilterQuery } from 'mongoose';
+import { PlayerActionEntry } from '../models/player-action-entry';
+
+@Injectable()
+export class ParseFiltersPipe implements PipeTransform {
+  constructor(private readonly playersService: PlayersService) {}
+
+  async transform(input: {
+    [key: string]: string;
+  }): Promise<FilterQuery<PlayerActionEntry>> {
+    if (isEmpty(input)) {
```

Comment:
If input is a string, why pull in a whole lib?

Reply:
I just wanted to filter out nulls, undefineds and empty objects... And we're using lodash anyway, so it doesn't really make any difference.